backend/src/services/item_info.py

```diff
@@ -29,7 +29,7 @@ def get_items(
         ids_part = f" WHERE A.id IN ({','.join([str(id) for id in ids])})"
     query = (
         f"SELECT {', '.join(fields)} FROM "
-        f"(SELECT *, GROUP_CONCAT(tag_value) as tags{search_query_part}"
+        f"(SELECT {', '.join([f for f in fields if f != 'tags'])}, GROUP_CONCAT(tag_value) as tags{search_query_part}"
```

Comment:
We need to explicitly specify the fields or else the group by will cause an issue (only_full_group_by = on is the default mode; we could set it to off, but better to just make revise this query).

Obviously, all of this code in item_info still needs to be refactored.